assets/app/views/create/fromimage.html

```diff
@@ -14,179 +14,194 @@
 
-                <div class="alert alert-info">
-                  <span class="pficon pficon-info"></span>
-                  <label>Note:</label> After creation, these settings can only be modified through the oc command.
-                </div>
-
-                <osc-form-section
-                  header="Routing" 
-                  about-title="Routing"
-                  about="Routing is a way to make your application publicly visible. Otherwise you may only be able to access your application by its IP address, if allowed by the system administrator."
-                  expand="true"
-                  can-toggle="false"
-                  class="first-section"
-                  >
-                  <div ng-hide="$parent.expand">
+                <div click-to-reveal link-text="Show advanced build and deployment options">
+                  <div class="form-group">
+                    <label for="gitref">Git Reference</label>
                     <div>
-                      <label>Create a route to the application: </label>
-                      <span>{{routing | yesNo}}</span>
-                    </div>
-                  </div>
-                  <div ng-show="$parent.expand">
-                    <div class="form-group">
-                      <label class="checkbox-inline">
-                        <input type="checkbox" ng-model="routing.include">
-                          Create a route to the application
-                          <span class="help action-inline">
-                            <i class="pficon pficon-help" data-toggle="tooltip" data-placement="right" data-original-title="Additional routes with alternative configuration options (security, TLS termination, etc) can be added using the oc command."></i>
-                          </span>
-                      </label>
+                      <input id="gitref" ng-model="buildConfig.gitRef" type="text"
+                            class="form-control form-control-md">
                     </div>
+                    <div class="help-block">Optional branch, tag, or commit.</div>
                   </div>
-                </osc-form-section> 
-
-                <!-- /routing -->
-
-                <osc-form-section 
-                  header="Deployment Configuration" 
-                  about-title="Deployment Configuration"
-                  about="Deployment configurations describe how your application is configured
-                    by the cluster and under what conditions it should be recreated (e.g. when the image changes)."
-                  expand="true"
-                  can-toggle="false"
-                  >
-                  <div ng-hide="$parent.expand">
-                    <h3>Autodeploy when</h3>
-                    <div>
-                      <label>New image is available: </label>
-                      <span>{{deploymentConfig.deployOnNewImage | yesNo}}</span>
-                    </div>
+                  <div class="form-group">
```

Comment:
there can (and will for some jboss images) be a sample git context dir annotation